web/src/lib/domain-logic/allowFormation.ts

```diff
@@ -1,36 +1,9 @@
-import {
-  BusinessPersona,
-  castPublicFilingLegalTypeToFormationType,
-  FormationLegalType,
-  PublicFilingLegalType,
-  publicFilingLegalTypes,
-} from "@businessnjgovnavigator/shared";
-
-export const allowFormation = (
-  legalStructureId: string | undefined,
-  persona: BusinessPersona | undefined
-): boolean => {
-  const featureFlagMap: Partial<Record<FormationLegalType, boolean>> = {
-    "foreign-limited-liability-company": process.env.FEATURE_BUSINESS_FLC === "true",
-    "foreign-limited-liability-partnership": process.env.FEATURE_BUSINESS_FLLP === "true",
-    "foreign-limited-partnership": process.env.FEATURE_BUSINESS_FLP === "true",
-    "foreign-s-corporation": process.env.FEATURE_BUSINESS_FCORP === "true",
-    "foreign-c-corporation": process.env.FEATURE_BUSINESS_FCORP === "true",
-    "foreign-nonprofit": process.env.FEATURE_BUSINESS_FNP === "true",
-    nonprofit: process.env.FEATURE_BUSINESS_NP === "true",
-  };
+import { PublicFilingLegalType, publicFilingLegalTypes } from "@businessnjgovnavigator/shared";
 
+export const allowFormation = (legalStructureId: string | undefined): boolean => {
   if (publicFilingLegalTypes.includes(legalStructureId as PublicFilingLegalType)) {
-    if (persona === "FOREIGN" || legalStructureId === "nonprofit") {
-      return (
-        featureFlagMap[
-          castPublicFilingLegalTypeToFormationType(legalStructureId as PublicFilingLegalType, persona)
-        ] ?? true
-      );
-    } else {
-      return true;
-    }
-  } else {
-    return false;
+    return true;
```

Comment:
This cannot be removed, because we don't support Foreign LP formation yet (and there are no plans to do so in the near future so far as I'm aware). So removing the `FLP` flag is going to break that experience. You could keep the feature flag in, or you could code against that situation explicitly, either way.

Reply:
That's my bad. I told him to do that.